### DISCUSSION TITLE
Is there a way to load a NextJS project into a VSCode Webview Extension?

### DISCUSSION THREAD

**Discussion by suesther** [vercel/next.js#53064] [Help]
### Summary

I am trying to configure [xstate-viz](https://github.com/statelyai/xstate-viz) so that I can view it as a webview in VSCode statically. Right now, running next build on xstate-viz creates all of these files:
<img width="421" alt="Screenshot 2023-07-22 at 10 37 15 PM" src="https://github.com/vercel/next.js/assets/135176059/ade7c6b0-22a8-4d38-8fe6-953b418568f4">

I want to be able to run the javascript in one of these files in order to get the visualization I want, and I started off with this template. If I add some html text to the body, something is rendered in the webview, but I want to render the xstate-viz state machine visualization. Could anyone give me some advice on which JS script I need to run? I have tried the main js file as well as other js files in the .next folder. Should I be running some combination of all of these folders in order to run?
<img width="811" alt="Screenshot 2023-07-22 at 10 38 11 PM" src="https://github.com/vercel/next.js/assets/135176059/927a6904-e7eb-4442-ae7d-5f5e164f07d6">

**Comment by hadihaider055** [comment]
I've faced a few issues recently, and built a boilerplate so that the Next.js project will automatically be built and exported to the extension.

https://github.com/hadihaider055/vscode-webview-boilerplate
